minifi/minifi-c2/minifi-c2-integration-tests/src/test/java/org/apache/nifi/minifi/c2/integration/test/AbstractTestUnsecure.java

```diff
@@ -36,7 +36,7 @@ public abstract class AbstractTestUnsecure {
     protected String c2Url;
 
     public static String getUnsecureConfigUrl(Container container) {
-        DockerPort dockerPort = container.port(10080);
+        DockerPort dockerPort = container.port(10090);
```

Comment:
It looks like there are multiple places where 10080 is used all of those could be updated as well. Could you please take a look?

Reply:
Thanks for spotting this, will update